tests/page-objects/AuthorDetailsPage.ts

```diff
@@ -0,0 +1,74 @@
+import { Selector, t } from 'testcafe';
+
+export class AuthorDetailsPage {
+    private readonly nextButton = Selector('.submission-wizard-next-button');
+
+    private readonly firstNameInput = Selector('.orcid-details__firstName');
+    private readonly lastNameInput = Selector('.orcid-details__lastName');
+    private readonly emailInput = Selector('.orcid-details__email');
+    private readonly institutionInput = Selector('.orcid-details__institution');
+
+    public async assertOnPage(): Promise<void> {
+        await t.expect(this.firstNameInput.visible).ok();
```

Comment:
We should probably assert on a `.author-step` class or something to the form to make the code easier to refactor in future. For Files we've just added a `.files-step` to the outer div

Reply:
Sensible. Was going to do that as part of another PR but equally valid. I'll get it done